spec/fabricators/category_fabricator.rb

```diff
@@ -10,23 +10,13 @@
   skip_category_definition false
 end
 
-Fabricator(:diff_category, from: :category) do
-  name "Different Category"
-  user
-end
-
-Fabricator(:happy_category, from: :category) do
-  name 'Happy Category'
-  slug 'happy'
-  user
-end
-
 Fabricator(:private_category, from: :category) do
   transient :group
 
-  name 'Private Category'
-  slug 'private'
+  name { sequence(:name) { |n| "Private Category #{n}" } }
```

Comment:
Why that change?

Reply:
Fabricating two `:private_category` in the same context would raise a 'duplicate name / slug' validation error.